### PR TITLE
[#1267] fix: Add missing headers and license

### PR DESCRIPTION
**Pull request by brandboat**
### What changes were proposed in this pull request?
add missing headers and license

### Why are the changes needed?
Fix: #1267

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
N/A (no code change, only add missing license)

### CODE REVIEW

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/service/MysqlService.java [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
```
Comment (FANNG1) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/service/MysqlService.java:2]: It's 2024 now,  happy new year!

Comment (qqqttt123) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/service/MysqlService.java:2]: No, this file is written in 2023.

**Review by qqqttt123 — APPROVED** [review]

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/service/MysqlService.java (hunk 1 above)
Comment (FANNG1) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/service/MysqlService.java:2]: oh

**Comment by qqqttt123** [comment]
Thanks @brandboat @brandboat , merged to master & branch 0.3.

**Pull request merged**